eosiojava/src/main/java/one/block/eosiojava/models/rpcProvider/Transaction.java

```diff
@@ -66,6 +70,10 @@
     @NotNull
     private List<String> transactionExtensions;
 
+    @SerializedName("context_free_data")
+    @NotNull
+    public ContextFreeData contextFreeData;
```

Comment:
In the EosJS implementation are they adding the context free data to the transaction model?  It did not seem like the context free data was supposed to get serialized as part of the Transaction model.  Let's discuss this approach.

Reply:
Yeah the CFD passes through with the transaction and then gets serialized separately: https://github.com/EOSIO/eosjs/blob/develop/src/eosjs-api.ts#L303